### PR TITLE
feat(component): creating blipLoading component

### PR DESCRIPTION
**Pull request by nicolasamaduro**
creating blipLoading component

### CODE REVIEW

File: src/components/blipLoading/blipLoading.component.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import angular from 'core/angular';
+import { BlipLoading } from 'blip-toolkit';
+import * as uuid from 'uuid';
+import { IComponentController} from 'angular';
+
+const BLIP_LOADING_PREFIX = 'blip-loading-';
+
+class BlipLoadingController implements IComponentController {
+    type: string;
```
Comment (samwx) [inline src/components/blipLoading/blipLoading.component.ts]: Use union type to declare wich loading type is possible: `type: 'dotted' | 'solid';`

Comment (nicolasamaduro) [reply src/components/blipLoading/blipLoading.component.ts]: nice! while I'm using it, does `'dotted` become the default value? If yes, I can remove `|| 'dotted'`
from here: 
` const instanceOptions = {
        type: this.type || 'dotted',
  };`
right?

Comment (samwx) [reply src/components/blipLoading/blipLoading.component.ts]: Nope! You can define default value like this: `type: 'dotted' | 'solid' = 'dotted'`;

**Pull request merged**